### PR TITLE
Feature/create product detail hung

### PR DESCRIPTION
**Pull request by vandangnhathung**
Product detail (description and overview product):
image
image

### CODE REVIEW

**Review by HaiDang1501 — COMMENTED** [review]
CSS

File: src/components/organisms/ProductDetail.jsx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                <h5 className="productDetail__description-title">
+                  Product description
+                </h5>
+                <p className="productDetail__description-content">
+                  A timeless design, with premium materials features as one of
+                  our most popular and iconic pieces. The dandy chair is perfect
+                  for any stylish living space with beech legs and lambskin
+                  leather upholstery.
+                </p>
+                <ul className="productDetail__description-fabric">
```
Comment (HaiDang1501) [inline src/components/organisms/ProductDetail.jsx:84]: CSS thẻ ul thụt vào 1 tí sẽ đẹp hơn

**Review by 19522515 — APPROVED** [review]

**Review by VKH2801 — APPROVED** [review]

**Review by PhamneyuT — APPROVED** [review]
Seems good

File: src/components/organisms/ProductDetail.jsx (hunk 1 above)
Comment (vandangnhathung) [reply src/components/organisms/ProductDetail.jsx:70]: done

**Pull request merged**